client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks/useStocktakeLineEdit.ts

```diff
@@ -35,6 +44,58 @@ export const useStocktakeLineEdit = (
       })
     );
 
+  const save = async () => {
+    let result;
+    try {
+      result = await mutateAsync(draftLines);
+    } catch (e) {
+      return { errors: errorMessage(e) };
+    }
+
+    const insertResults = result.batchStocktake?.insertStocktakeLines || [];
+    const updateResults = result.batchStocktake?.updateStocktakeLines || [];
+
+    let errorMessagesMap: { [key: string]: string } = {};
```

Comment:
:nit not reassigned
```suggestion
    const errorMessagesMap: { [key: string]: string } = {};
```

Reply:
cfe481f9f56dabfd7dcef924b1bc70caf8a818b6